SPECS/rdma-core/rdma-core.spec

```diff
@@ -20,13 +20,23 @@ Patch1: 0001-kernel-boot-Do-not-perform-device-rename-on-OPA-devi.patch
 # 32-bit arm is missing required arch-specific memory barriers,
 ExcludeArch: %{arm}
 
+%global azl3 1
 BuildRequires: binutils
 BuildRequires: cmake >= 2.8.11
 BuildRequires: gcc
 BuildRequires: pkg-config
 BuildRequires: pkgconfig(libnl-3.0)
 BuildRequires: pkgconfig(libnl-route-3.0)
+# Disable pyverbs for azl3, as pyverbs cannot build with cython > 3
```

Comment:
I appreciate the `if-else` rationale here :)

Reply:
Thanks. This is what works for us. 
Also, Fedora 40 onwards does not build pyverbs either - so we are not an outlier w.r.t other distributions.